apps/trackers/bugzilla/query.py

```diff
@@ -91,7 +91,24 @@ def generate_cc(self):
         """
         generate query for CC list
         """
-        # TODO CC list module
+
+        # NOTE: AffectCCBuilder doesn't implement private_tracker_cc (as of April 2024).
+        #       This is not a problem because we will file BZ trackers only for public
+        #       projects (probably only Fedora).
+        if self.tracker.external_system_id:
+            # Add CCs only on creation.
+            return
+
+        affect_cc_builder = AffectCCBuilder(
+            self.tracker.affects.first(),
```

Comment:
I am not sure we agree on supporting multi-flaw trackers (another comment) but if so we should adjust this piece of code accordingly.

Reply:
Yes, will fix by creating per-affect CC lists and then merging them.